koin-projects/examples/android-mvp/src/main/kotlin/fr/ekito/myweatherapp/di/app_module.kt

```diff
@@ -29,7 +29,7 @@ val weatherAppModule = module {
     factory<WeatherListContract.Presenter> { WeatherListPresenter(get(), get()) }
 
     // Presenter for Detail View
-    scope<DetailActivity> {
```

Comment:
Why change the scope here? Presenter is tied to the DetailActivity's scope.

Reply:
Sorry, fixed